### PR TITLE
CNV#40186: Placeholder file for 4.17 RN

### PR DESCRIPTION
**Pull request by danielclowers**
Version(s):
main only

Issue:
https://issues.redhat.com/browse/CNV-40186

Link to docs preview:
https://79783--ocpdocs-pr.netlify.app/openshift-enterprise/latest/virt/release_notes/virt-release-notes-placeholder

QE review:
NA

Additional information:

### CODE REVIEW

**Review by lpettyjo — COMMENTED** [review]
LGTM!

File: virt/release_notes/virt-4-17-release-notes.adoc [hunk 1]
```diff
@@ -0,0 +1,11 @@
+:_content-type: ASSEMBLY
+[id="virt-4-17-release-notes"]
+= {VirtProductName} release notes
+include::_attributes/common-attributes.adoc[]
+:context: virt-4-17-release-notes
+toc::[]
```
Comment (Srivaralakshmi) [inline virt/release_notes/virt-4-17-release-notes.adoc:6]:
```suggestion

toc::[]

```
Leave a line space before  L6, as per the ocp docs guidelines, before toc:[]

**Review by Srivaralakshmi — COMMENTED** [review]
@danielclowers I have left a comment, ptal at it and address. Thank you!

File: virt/release_notes/virt-4-17-release-notes.adoc [hunk 2]
```diff
@@ -0,0 +1,12 @@
+:_content-type: ASSEMBLY
```
Comment (bscott-rh) [inline virt/release_notes/virt-4-17-release-notes.adoc:1]:
```suggestion
:_mod-docs-content-type: ASSEMBLY
```

**Comment by danielclowers** [comment]
It was my mistake to request this to be CP'd to 4.17. It making it into the main branch was all that was needed. @bscott-rh can you remove the enterprise-4.17 label from this PR if you don't mind?

**Comment by bscott-rh** [comment]
> It was my mistake to request this to be CP'd to 4.17. It making it into the main branch was all that was needed. @bscott-rh can you remove the enterprise-4.17 label from this PR if you don't mind?

No problem. you may want to remove it from the PR subject just for posterity.

**Pull request merged**